src/kvilib/system/KviEnvironment.h

```diff
@@ -32,51 +32,67 @@
 
 #include "kvi_settings.h"
 
+#include <QString>
+
 #include <stdlib.h>
 
 #if defined(COMPILE_ON_WINDOWS) || defined(COMPILE_ON_MINGW)
 #include <windows.h>
+#define MAX_ENV_SIZE 32767
 #endif
 
 namespace KviEnvironment
 {
 
-	/**
-	* \brief Gets environment variable
-	* \param name The name of the variable to get
-	* \return char *
-	*/
-	inline char * getVariable(const char * name)
+#if defined(COMPILE_ON_WINDOWS) || defined(COMPILE_ON_MINGW)
+	inline QString getVariable(QString szName)
```

Comment:
The `QString`s passed should be passed as `const QString&` to avoid an unnecessary copy.